src/components/LatestVersion/index.tsx

```diff
@@ -1,53 +1,52 @@
-import React, { useState, useEffect } from "react";
-import Link from "@docusaurus/Link";
-import moment from "moment";
+import React, { useState, useEffect } from "react"
+import Link from "@docusaurus/Link"
+import moment from "moment"
 
 // Bringing in the styles from the `pages/index.module.css` file to not have to duplicate the styles (nothing is new in this component)
-import styles from "../../pages/index.module.css";
-import * as Arrow from "@site/static/img/arrow.svg";
+import styles from "../../pages/index.module.css"
```

Comment:
A lot of this looks like conflicting formatter churn. Can we either:

a. avoid committing the code style changes (I'm assuming this means disable your default formatting settings that you may have in your editor for this workspace)
b. add configuration to the project to use for formatting

I personally prefer option A, because I don't want to maintain more formatting config than I have to, but low GAFO.